algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -200,6 +200,20 @@ case class QTree[A](
       (0L, 0L)
     }
   }
+  /**
+   * For a given value and given quantiles list, compute which quantile it fails to in this QTree
+   * @param value The specified value to compute quantile.
+   * @param quantiles The specified quantiles list, need to be sorted.
+   */
+  def getValueQuantile(value: Double, quantiles: Seq[Double]): (Double, Double) = {
+    val quantilePoints = quantiles.map { q: Double =>
+      val (lower, upper) = quantileBounds(q)
+      (lower, upper, q)
+    }
```

Comment:
can you add a few comments:

1. how do you know there is only one item in this list?
2. I am confused about the algorithm have `>=` and `1.0` in both cases. Can you comment a bit?

It seems to me that you want something like:
`quantilePoints.filter { case @b(l, u) => (l <= value && value <= u) }`
Why is that not what we want?